recipes-smt/yices2/yices2_git.bb

```diff
@@ -0,0 +1,23 @@
+DESCRIPTION = "Yices 2 is a solver for Satisfiability Modulo Theories (SMT) problems."
+HOMEPAGE = "https://yices.csl.sri.com/"
+LICENSE = "GPLv3"
+SECTION = "devel/verilog"
+
+LIC_FILES_CHKSUM = "file://LICENSE.txt;md5=d32239bcb673463ab874e80d47fae504"
+SRC_URI = "git://github.com/SRI-CSL/yices2;protocol=https"
+SRCREV = "98fa2d882d83d32a07d3b8b2c562819e0e0babd0"
+
+S = "${WORKDIR}/git"
+
+PV = "2.6.1+git${SRCPV}"
+
+inherit autotools autotools-brokensep
+
+DEPENDS = " \
+	gperf-native \
+	gmp \
+	"
+
+EXTRA_OEMAKE += "YICES_MAKE_INCLUDE=configs/make.include.${TARGET_SYS} YICES_MODE=debug"
```

Comment:
Is the `YICES_MODE=debug` required? I did not look into what is does specifically, though I suspect you may have added it to resolve `already-stripped` QA warnings/errors.

Reply:
Yes, YICES_MODE=debug was added in order to resolve the already-stripped QA warnings/errors. By default, YICES_MODE is set to "release" which will strip all the binaries.

The modes are described in https://github.com/SRI-CSL/yices2/blob/master/doc/COMPILING , section "BUILD COMMANDS".

Reply:
That document states that debug disables optimizations, which I imagine would be useful. Is it possible to trick the makefile to skip stripping similar to how yosys is tricked by swapping swapping `strip` for `echo`? (https://github.com/nathanrossi/meta-hdl/blob/master/recipes-synthesis/yosys/yosys_git.bb#L61)

Reply:
Thanks for the suggestion! This is indeed a better approach. I have updated the commit.